common/views/components/TabNav/TabNav.tsx

```diff
@@ -1,42 +1,47 @@
-import { ComponentType } from 'react';
+import { FunctionComponent } from 'react';
 import styled from 'styled-components';
-import NextLink from 'next/link';
-import { TextLink } from '../../../model/text-links';
+// import { NextLinkType } from '@weco/common/model/next-link-type';
+import Space from '../styled/Space';
 import { font, classNames } from '../../../utils/classnames';
-import Space, { SpaceComponentProps } from '../styled/Space';
 
-type SelectableTextLink = TextLink & {
+type SelectableTextLink = {
+  id: string;
+  text: string;
+  // TODO we probably want anchors here so people can share the url/go back to the correct section?
+  // link: NextLinkType;
   selected: boolean;
-  onClick?: (event: SyntheticEvent<HTMLAnchorElement>) => void;
+  color?: string;
+  onClick: (id: string) => void;
 };
 
 type Props = {
   items: SelectableTextLink[];
+  color?: string;
 };
 
-const NavItemInner: ComponentType<SpaceComponentProps> = styled(Space).attrs(
-  props => ({
+type NavItemInnerProps = {
+  selected: boolean;
+};
+
+// TODO do we want to make the border colour dynamic or not?
+const NavItemInner = styled(Space).attrs<NavItemInnerProps>(props => {
+  return {
     className: classNames({
       selected: props.selected,
       block: true,
       relative: true,
     }),
-  })
-)`
+  };
+})<NavItemInnerProps>`
   z-index: 1;
-  padding: 0 0.3em;
+  padding: 0 0.3em 1.5em;
+  border-bottom: 0 solid ${({ color, theme }) =>
+    theme.color(color, 'dark') || theme.color('black')};
+  transition: width 200ms ease;
```

Comment:
There's a `transitionProperties` in the theme that is equal to `150ms ease` (I'm not convinced that all of our transitions should be `150ms ease` but maybe we could have a system for choosing a time/easing function based on the type of transition).

Reply:
Again this came from older code and I just had a look through the project and there seems to be a lot of variants of it. We could suggest slow-default-fast times in the theme?